src/youtube-player/youtube-player.ts

```diff
@@ -119,6 +119,18 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
   private _width = DEFAULT_PLAYER_WIDTH;
   private _widthObs = new EventEmitter<number>();
 
+  /** PlayerVars for the YouTube player */
+  @Input()
+  get playerVars(): YT.PlayerVars | undefined {
```

Comment:
It might be better to have individual inputs for the different `playerVars` properties, rather than the entire object. We've been avoiding adding inputs where you pass in a configuration object, because change detection won't pick up changes to individual properties. What do you think @jelbourn?

Reply:
I would also lean in that direction; it's the path we took for the google-maps component, since it makes the settings align more with Angular's framework idioms

Reply:
How about using the same api and within `ngOnChanges` => detecting "special" options and updating change detection settings accordingly? 

Either way, I am looking forward to making use of this soon.

Also: Is there any workaround right now to customise the player?

Reply:
> Also: Is there any workaround right now to customise the player?

I take the silence as a no. ;)

Reply:
Sorry, I've been pretty busy with everything at work, I should be able to begin work on this again next week.